web/js/modules/votemain.js

```diff
@@ -0,0 +1,45 @@
+import { RankedPoll } from "./ranked-poll.js";
+import { loadWorker } from "./lib.js";
+import { VolumeManager } from "./volume.js";
+import { TokenManager } from "./token.js";
+// ranked poll API
+let activePoll = null;
+
```

Comment:
The only thing missing compared to `main.js` seems to be the loading of the countdown worker. That could be `if`ed, or just let it load.